block_my_feedback.php

```diff
@@ -226,7 +226,7 @@ public function is_course_current(stdClass $course): bool {
      */
     public function duedate_in_range(int $duedate): ?int {
         // Only show dates within a month.
-        $past = strtotime('-1 month');
+        $past = strtotime('-2 month');
```

Comment:
Personally I'd not use single-use variables, the comments make it clear what's going on. We could instead have:

```
        // Do not show dates outside this range.
        if (($duedate < strtotime('-2 month')) || ($duedate > strtotime('+1 month'))) {
            return false;
        }
```